recruiter/views.py

```diff
@@ -74,4 +57,22 @@ def recruiterRegister(request):
 
 
 def recruiter_created_successfully(request):
+    jobboard.views.add_navbar_links_to_context(request,)
     return render(request, 'recruiter/recruiter_created_successfully.html')
+
+
+def create_new_job_form(request):
```

Comment:
Why have this in `recruiter.views` and not in `jobbard.views`?

Reply:
@ifireball since it's a feature that only a recruiter has I thought it would make more sense to put it in the recruiter.views

Reply:
The models, the forms that edit them, the views that show them and the tests for them typically all go in the same app. Where would you intuitively look for the view that creates new Job objects?

Reply:
I guess your right. to be honest, intuitively it made sense to me to put it in the recruiter views since it is a recruiter feature. But now I understand your point and it does make more sense to me to put it in the jobboard views. I will fix it now and update the PR.

Reply:
@ifireball fixed it and moved the function and the form to the jobboard app